.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,4 +1,33 @@
-
+<!--
+Thanks for sending a pull request!  Here are some tips for you:
+  1. If this is your first time, please read our [contributor guidelines](https://github.com/elyra-ai/community/blob/master/contributing.md)
+    1.1 Please follow the [7 rules for a great commit message](https://github.com/elyra-ai/community/blob/master/contributing.md#creating-a-pull-request)
+  2. If the PR is unfinished, leave it as 'DRAFT', add '[WIP]' in your PR title, e.g., '[WIP] Your PR title ...' or use the 'status:Work in Progress' label.
+  3. Be sure to keep the PR description updated to reflect all changes.
+  4. Please write your PR title to summarize what this PR proposes.
+  5. If this PR involves UI changes, please provide a screen capture (before/after) for a faster review.
+  6. Remember to add 'Fixes #ISSUE_NUMBER' to get the issue properly closed when the PR is merged
```

Comment:
```suggestion
  6. Remember to add 'Fixes #ISSUE_NUMBER' to get the issue properly closed when the PR is merged.
```

Reply:
Perhaps we should add a reminder to check whether documentation updates are required?